tests/subsys/tracing/testcase.yaml

```diff
@@ -0,0 +1,12 @@
+tests:
+  tracing.transport:
+    tags: tracing_disabled
```

Comment:
what is the purpose of this? What are you testing?

Reply:
as above reply, This is for the case when tracing functions disabled, if tracing functions are disabled, we shouldn't find any tracing data in the backend.